quattor/types/hardware.pan

```diff
@@ -175,6 +175,21 @@ type structure_sysloc = {
 };
 
 @documentation{
+    System benchmark results
+    benchmarks is used to hold the performance benchmark for the machine 
+    i.e. HEPSpec06 score
+    this might be used to scale things such as the wall time
+    example of using this might be : 
+    variable CONDOR_WN_SCALING_FACTOR = value('/hardware/benchmarks/hepspec06') 
+          / ( 4 * get_num_of_cores()) ;
+}
+type structure_benchmark = {
+    "hepspec06" ? double
+    "HPL" ? double
+    "stream" ? double
```

Comment:
Is the stream definition too simplistic? Should it be named `stream-triad` or similar?